presto-orc/src/test/java/com/facebook/presto/orc/OrcTester.java

```diff
@@ -645,7 +646,7 @@ private static DataSize writeOrcColumnPresto(File outputFile, Format format, Com
         writer.write(new Page(blockBuilder.build()));
         writer.close();
         writer.validate(new FileOrcDataSource(outputFile, new DataSize(1, MEGABYTE), new DataSize(1, MEGABYTE), new DataSize(1, MEGABYTE), true));
-        return new DataSize(output.size(), Unit.BYTE);
+        return new DataSize(new OutputStreamSliceOutput(outputStream).longSize(), Unit.BYTE);
```

Comment:
This will return zero because the stream was never used.  The return value is never used, so just remove it.